arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -0,0 +1,495 @@
+                    "description": "Gets the type of Import/Export opertion being performed"
+                },
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ExportRequestParameters"
+                }
+            },
+            "description": "Represents the properties for an import operation"
+        },
+        "ImportExtensionRequestParameters": {
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "Gets the name of the extenstion."
+                },
+                "type": {
+                    "type": "string",
+                    "description": "Gets the type of the extenstion."
+                },
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ImportExtensionProperties",
+                    "description": "Represents the properties of the resource."
+                }
+            },
+            "description": "Import Azure SQL Database parameters."
+        },
+        "ImportExportOperationStatusResponse": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ImportExportOperationStatusResponseProperties",
+                    "description": "Gets or sets the properties of the response."
+                },
+                "id": {
+                    "type": "string",
+                    "description": "Gets or sets the id returned from the server."
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Gets or sets the name returned from the server."
+                },
+                "type": {
+                    "type": "string",
+                    "description": "Gets or sets the type returned from the server."
+                }
+            },
+            "description": "Response for Azure Sql Import/Export Status operation."
+        },
+        "ImportExportOperationStatusResponseProperties": {
+            "properties": {
+                "requestType": {
+                    "type": "string",
+                    "description": "Gets or sets the request type of the operation."
+                },
+                "requestId": {
+                    "type": "string",
+                    "format": "uuid",
+                    "description": "Gets or sets the request type of the operation."
+                },
+                "serverName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL Server."
+                },
+                "databaseName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL Database."
+                },
+                "status": {
```

Comment:
Fixed.

---
In reply to: [80154726](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80154726) [](ancestors = 80154726)